pandas/core/reshape/merge.py

```diff
@@ -598,6 +602,11 @@ def merge_asof(
     return op.get_result()
 
 
+def _chunks(lst, n):
```

Comment:
Can this be typed?

Reply:
done, used collections.abc.Sized for lst, since Iterable would erroneously include generators, and Sequence didn't include DataFrames